src/constants.py

```diff
@@ -5,22 +5,30 @@
 
 KYUUBI_CONTAINER_NAME = "kyuubi"
 KYUUBI_SERVICE_NAME = "kyuubi"
+
+# Database related literals
 METASTORE_DATABASE_NAME = "hivemetastore"
 AUTHENTICATION_DATABASE_NAME = "auth_db"
 AUTHENTICATION_TABLE_NAME = "kyuubi_users"
 POSTGRESQL_DEFAULT_DATABASE = "postgres"
 
+# Relation names
 S3_INTEGRATOR_REL = "s3-credentials"
 POSTGRESQL_METASTORE_DB_REL = "metastore-db"
 POSTGRESQL_AUTH_DB_REL = "auth-db"
 SPARK_SERVICE_ACCOUNT_REL = "spark-service-account"
+ZOOKEEPER_REL = "zookeeper"
+KYUUBI_CLIENT_RELATION_NAME = "jdbc"
 
+# Literals related to K8s
 NAMESPACE_CONFIG_NAME = "namespace"
 SERVICE_ACCOUNT_CONFIG_NAME = "service-account"
 
+# Literals related to Kyuubi
 JDBC_PORT = 10009
-
 KYUUBI_OCI_IMAGE = "ghcr.io/canonical/charmed-spark-kyuubi:3.4-22.04_edge"
-
 DEFAULT_ADMIN_USERNAME = "admin"
-KYUUBI_CLIENT_RELATION_NAME = "jdbc"
+
+# Zookeeper literals
+HA_ZNODE_NAME = "/kyuubi"
+HA_ZNODE_NAME_TEMP = "/kyuubi-temp"
```

Comment:
**suggestion** I would not put tmp constants (which are also workaround) among the actual constants. Given it is a workaround, we could just hard code this in the request (or even randomly generate that) :D 

anyhow, it is a fairly small point

Reply:
I agree.